PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,5 @@
+#### What's this PR do?
```

Comment:
For whatever reason, this PR Template never worked for us on the `develop` branch. Maybe it'll work on master after this merge